geo/src/lib.rs

```diff
@@ -123,12 +123,13 @@ pub mod prelude {
     pub use crate::algorithm::vincenty_length::VincentyLength;
 }
 
-/// A common floating point trait used for geo algorithms.
+/// A common numeric trait used for geo algorithms.
 ///
 /// Different numeric types have different tradeoffs. `geo` strives to utilize generics to allow
 /// users to choose their numeric types. If you are writing a function which you'd like to be
-/// generic over the floating point types supported by geo, you probably want to constrain
-/// your function input to GeoFloat.
+/// generic over all the numeric types supported by geo, you probably want to constraint
+/// your function input to `GeoFloat`. For methods which work for integers, and not just floating
+/// point, see [`GeoNum`](trait.GeoFloat.html).
```

Comment:
Don't need to change it, but FYI I think you should be able to just do:

```
/// point, see [`GeoNum`].
```

and rustdoc will make it link to the appropriate docs

Reply:
Done. Thanks!